templates/base.jinja2

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Learning Journal</title>
+  <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,800' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" type="text/css" href="static/css/style.css">
+</head>
+<body>
+  <header class="container">
+    <h1>Scott's Learning Journal</h1>
+    <nav>
+      <ul>
+        {% if not request.authenticated_userid %}
+          <li><a href="{{ request.route_url('login') }}">Login</a></li>
+          <li><a href="{{ request.route_url('home') }}">Home</a></li>
+        {% else %}
+          <li><a href="{{ request.route_url('logout') }}">Logout</a></li>
+          <li><a href="{{ request.route_url('home') }}">Home</a></li>
+          <li><a href="{{ request.route_url('new-entry') }}">New Entry</a></li>
```

Comment:
Do you need to repeat the home button in both clauses?  Might you organize things differently to make a `DRY`er template?